docs/advanced-features/sso-permissions.md

```diff
@@ -0,0 +1,63 @@
+# SSO permissions mapping
+
+> This feature requires Quilt stack version 1.54.0 or higher
+
+## Overview
+
+This feature allows Quilt admin to configure what roles and admin flag value are
+assigned to the user who logs in via SSO given on the user's ID token claims.
+
+The configuration can be initially set with admin `quilt3.admin.sso_config.set()`
+and subsequently can be update with admin UI as well.
+
+![admin UI for setting SSO permissions mapping](../imgs/admin-sso-config.png)
+
+> Note: Roles used by configuration can't be removed or renamed.
+
+> Note: User who sets configuration last can't be revoked admin flag.
```

Comment:
but currently this user still can be disabled, that doesn't look right, does it?

Reply:
That is less than ideal, but not a showstopper.  Worth documenting, though.

Reply:
Ever or just in the edit AFTER they set the configuration. Agree this needs to be documented.

Reply:
I read the new string and I think this is permanent. It seems like a really odd side-effect. So even if someone sets the config after they do you can't revoke the old admin? That doesn't sound right indeed.

Reply:
@sir-sigurd Can a new admin create a mapping that disables the Admin Flag for the previous Admin?

Reply:
> So even if someone sets the config after they do you can't revoke the old admin? 

admin flag can't be revoked only from use who set config ***last***

my concern was that config uploader still can be disabled, so AFAICT you still end up without a single active admin in the stack under some circumstances